### PR TITLE
feat: [CDS-52408]: Google Cloud Storage artifact trigger UI changes

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactConnector.tsx [hunk 1]
```diff
@@ -179,12 +181,14 @@ export function ArtifactConnector(props: StepProps<ConnectorConfigDTO> & Artifac
               </Layout.Horizontal>
             </div>
             <Layout.Horizontal spacing="medium">
-              <Button
-                variation={ButtonVariation.SECONDARY}
-                text={getString('back')}
-                icon="chevron-left"
-                onClick={() => previousStep?.(prevStepData)}
-              />
+              {showArtifactSelectionStep && (
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactConnector.tsx:184]: Is this component used only in triggers?

Comment (pankajsingh-harness) [reply src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactConnector.tsx:184]: This is used only in artifacts and trigger.

Comment (pankajsingh-harness) [reply src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactConnector.tsx:184]: This is just used in `ArtifactWizard` component. Now came `ArtifactWizard` component is used in artifacts and triggers with this change along with to show or hide the artifact selection step

**Review by rameshx — APPROVED** [review]

File: src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactUtils.ts [hunk 2]
```diff
@@ -44,63 +43,6 @@ export const resetTag = (formik: FormikValues): void => {
     formik.setFieldValue('tag', '')
 }
 
-export const helperTextData = (
```
Comment (AshwinBhatkal-Harness) [inline src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactUtils.ts:47]: Is this info moved somewhere else?

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactUtils.ts (hunk 2 above)
Comment (pankajsingh-harness) [reply src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactUtils.ts:47]: This was not not used any where. IDE was not showing this as unused due to export. So removed this.

**Pull request merged**